src/main/scala/beam/agentsim/agents/modalbehaviors/ChoosesMode.scala

```diff
@@ -1146,9 +1144,9 @@ object ChoosesMode {
       parkingResponse = if (withParking) {
         None
       } else {
-        Some(ParkingInquiryResponse(ParkingStall.lastResortStall(), 0))
+        Some(ParkingInquiryResponse(ParkingStall.defaultStall(new Coord(0.0, 0.0)), 0))
```

Comment:
why is (0,0) better than a random stall location? i think it would be the same amount of work to re-map to a link in the network, and (0,0) makes assumptions about the coordinates of the running scenario (like, what about when (0,0) is a real location on the user's network?).

i should probably also mention my reasoning behind lastResortStall. it was suggested to me that it is acceptable to depend on connecting a random coordinate back to the scenario's road network. that, and the fact that the map is not in scope here, made it seem to be the best solution.

Reply:
At _this_ point in the code, it means something different yet again, unfortunately. This stall is actually _never_ used, it is just a dummy object that we need to get through the logic of this class in a certain way. Bad, bad, bad, I know.